generators/pythonv2/codegen/src/ast/__test__/__snapshots__/PythonFile.test.ts.snap

```diff
@@ -27,6 +27,22 @@ class DeeplyNestedTestClass(DeepClass):
 "
 `;
 
+exports[`PythonFile > Add a class with a relative import and alias 1`] = `
+"from ...test_module.test.sibling_dir import SiblingClass
+
+class TestClassWithRelativeAlias(SiblingClass as AliasedSibling):
```

Comment:
[blocking] this looks incorrect I don't think python supports inline `as` aliasing within the base class inheritance

Reply:
Ah true good catch. Will fix in a follow-up.